samples/graph-notification-team-channel/csharp/README.md

```diff
@@ -0,0 +1,104 @@
+---
+page_type: sample
+description: This is a sample application which demonstrates use of team/Channel subscription that will post notifications when user create/edit/delete.
+products:
+- office-teams
+- office
+- office-365
+languages:
+- csharp
+extensions:
+contentType: samples
+createdDate: "25-08-2022 04:30:15"
+---
+
+# This is a sample application that shows the usage of Graph API Notifications related to team/channels tab.
+
+This is a sample application which demonstrates use of Team/Channel subscription that will post notifications when user create/edit/delete channel.
```

Comment:
This is a sample application which demonstrates use of Team/Channel subscription that will post notifications when user create/edit/delete team/channel.

Reply:
fixed